src/main/java/crud/dao/impl/DriverDaoJdBcImpl.java

```diff
@@ -0,0 +1,113 @@
+package crud.dao.impl;
+
+import crud.dao.abstraction.DriverDao;
+import crud.lib.DaoImpl;
+import crud.lib.DataBaseException;
+import crud.model.Driver;
+import crud.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@DaoImpl
+public class DriverDaoJdBcImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String query = "INSERT INTO drivers (driver_name, driver_license_number) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            setUpdate(statement, driver).executeUpdate();
+            ResultSet set = statement.getGeneratedKeys();
+            if (set.next()) {
+                driver.setId(set.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException throwables) {
+            throw new DataBaseException("Couldn't create insert "
+                    + driver + " into driversDB. ", throwables);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String query = "SELECT driver_id, driver_name, driver_license_number FROM drivers "
+                + "WHERE driver_id = " + id + " AND deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement statement = connection.createStatement()) {
```

Comment:
Just use PreparedStatement Because it will be more secure
```suggestion
        String query = "SELECT driver_id, driver_name, driver_license_number FROM drivers "
                + "WHERE driver_id = ? AND deleted = FALSE";
        try (Connection connection = ConnectionUtil.getConnection();
                Statement statement = connection.createStatement()) {
```